Userland/Libraries/LibWeb/CSS/Clip.cpp

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
These changes to the computed values and style properties and such are not strictly related to parsing. I think you can parse the style value without these (which could be in a later commit).